### DISCUSSION TITLE
Submariner for Azure AKS

### DISCUSSION THREAD

**Discussion by zohebs341** [submariner-io/submariner#2877] [General]
Hi Team,

Did someone implement or use Submariner for Azure AKS?

I am very much interested in doing POCs for Azure AKS.

**Comment by yboaron** [comment]
Hi @zohebs341 ,

If the CNI is supported by Submariner and the firewall requirements (Intra cluster udp port 4800) for Submariner are supported, theoretically you should be able to deploy Submariner on AKS using LoadBalancer service.

Which CNI are you using?

We have successfully deployed Submariner on an Azure ARO managed service using LoadBalancer service.

You can check [1] for a reference on deploying Submariner using LoadBalancer service.

[1]
https://submariner.io/getting-started/quickstart/openshift/gcp-lb/
